docs/recipes/auth/README.md

```diff
@@ -0,0 +1,96 @@
+# Running Lighthouse on Authenticated Pages with Puppeteer
+
+If you just want to view the code for using Lighthouse with Puppeteer, see [example-lh-auth.js](./example-lh-auth.js).
+
+## The Example Site
+
+There are two pages on the site:
+
+1. `/` - the homepage
+2. `/dashboard`
+
+The homepage shows the login form, but only to users that are not signed in.
+
+The dashboard shows a secret to users that are logged in, but shows an error to users that are not.
+
+The server responds with different HTML for each of these pages and session states, so there are four different pages that must have passable Lighthouse SEO scores.
+
+(Optional) To run the server:
+```sh
+# be in root lighthouse directory
+yarn # install global project deps
+cd docs/auth
+yarn # install deps related to just this documentation
+yarn start # start the server on http://localhost:8000
```

Comment:
once the server is started.. seems like we should also document running `node example-lh-auth.js`..

```sh
# ...
node example-lh-auth.js # login via puppeteer and run lighthouse
```

Reply:
hmm since you have to read the whole page to see your mention of example-lh-auth i worry that people might try to recreate everything before realizing that we actually provide it.

how about after your `yarn start` ...

    Now that the server is started, let's login with Puppeteer and then run Lighthouse:
    ```sh
    node example-lh-auth.js
    ```
    What does this do?  Read on....